scripts/plaid.conf.js

```diff
@@ -6,11 +6,20 @@ const { isProd } = require('@gera2ld/plaid/util');
  * - value.html: options object passed to HtmlWebpackPlugin.
  * - value.html.inlineSource: if true, JS and CSS files will be inlined in HTML.
  */
+const injectTo = item => {
+  if ([
+    '../browser.js',
+    '../common.js',
+  ].includes(item.attributes.src)) {
+    return 'head';
```

Comment:
After I **manually** reposition the tags in html file it indeed solves the FOUC and the white frame flashing, but `yarn build` doesn't place the script tags correctly, they're still inside `<body>`. Same for `yarn dev`.